modules/custom/az_news/config/install/core.entity_view_display.node.az_news.az_small_row.yml

```diff
@@ -0,0 +1,227 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.node.az_small_row
+    - field.field.node.az_news.field_az_attachments
+    - field.field.node.az_news.field_az_body
+    - field.field.node.az_news.field_az_byline
+    - field.field.node.az_news.field_az_contacts
+    - field.field.node.az_news.field_az_enterprise_attributes
+    - field.field.node.az_news.field_az_expiration_date
+    - field.field.node.az_news.field_az_featured_image_as_thumb
+    - field.field.node.az_news.field_az_link
+    - field.field.node.az_news.field_az_main_content
+    - field.field.node.az_news.field_az_media_image
+    - field.field.node.az_news.field_az_media_thumbnail_image
+    - field.field.node.az_news.field_az_news_tags
+    - field.field.node.az_news.field_az_published
+    - field.field.node.az_news.field_az_short_title
+    - field.field.node.az_news.field_az_subheading
+    - field.field.node.az_news.field_az_summary
+    - node.type.az_news
+  module:
+    - field_group
+    - smart_title
+    - user
+third_party_settings:
+  field_group:
+    group_row:
+      children:
+        - group_media
+        - group_text_column
+      label: Row
+      parent_name: group_link
+      region: content
+      weight: 0
+      format_type: html_element
+      format_settings:
+        classes: row
+        show_empty_fields: false
+        id: ''
+        element: div
+        show_label: false
+        label_element: h3
+        label_element_classes: ''
+        attributes: ''
+        effect: none
+        speed: fast
+    group_text_column:
+      children:
+        - group_heading
+        - group_read_more
+      label: 'Text Column'
+      parent_name: group_row
+      region: content
+      weight: 9
+      format_type: html_element
+      format_settings:
+        classes: 'col d-flex flex-column my-auto pb-3'
```

Comment:
```suggestion
        classes: 'col d-flex flex-column mt-0 mb-auto pb-3'
```

Reply:
Aligns text with top of thumbnail.

Reply:
This is a request from central marketing to have it aligned to the center of the image since it does not encompass other content beyond the title.